google-cloud-bigtable-stats/src/main/java/com/google/cloud/bigtable/stats/BuiltinMeasureConstants.java

```diff
@@ -29,6 +29,20 @@ class BuiltinMeasureConstants {
   static final TagKey ZONE = TagKey.create("zone");
   static final TagKey CLIENT_UID = TagKey.create("client_uid");
 
+  // Monitored resource TagKeys for gce_instance
+  static final TagKey GCE_PROJECT_ID = TagKey.create("gce_project_id");
```

Comment:
Do we need to define these keys here? Or this logic could stay in the exporter when we're constructing the detected monitored resource?